README.md

```diff
@@ -180,3 +196,32 @@ scripts if requested.
 
 There are several tools trying to solve similar problems as Jujutsu. See
 [related work](https://martinvonz.github.io/jj/latest/related-work) for details.
+
+## Contributing
+
+We welcome outside contributions, and there's plenty of things to do, so
+don't be shy. Please ask if you want a pointer on something you can help with,
+and hopefully we can all figure something out.
+
+We do have [a few policies and
+suggestions](https://martinvonz.github.io/jj/prerelease/contributing/)
+for contributors. The broad TL;DR:
```

Comment:
I'm on the fence about adding the TL;DR. It requires syncing with the contributing doc. If we keep it, perhaps call it "The broad TL;DR of the mandatory policies"?

Aside: This also makes me think that maybe we should split the contributing doc into two (the policies and the suggestions).

Reply:
TBH, I doubt it's a huge deal, unless we expect the policies to change substantially? The wording may change, sure. But the broad strokes? FWIW, I sort of treat the README as just a glorified "sort of HTML file" that is used to give basic guidance for the project, so I think including little bits like this is fair and gives people a reasonable impression about how things work. And enough people have hit the CLA thing and disliked it that I think it should be called out explicitly, so I just added a little extra. Not any more burdensome to synchronize than a lot of the other minor doc details in the long run, I think.

Reply:
Your reasoning makes sense. I'm still on the fence, looking at both sides of it and being unsure where the grass is greener. Ultimately, I think it'd be OK either way.

I'll punt this to @martinvonz, we can go with whatever he prefers.

Reply:
I'm fine with it as is. It doesn't seem like much of maintenance burden to me.